### PR TITLE
Ansible inventory generation from Rundeck nodes

### PR DESCRIPTION
**Pull request by erikasm**
In our use case we generate Ansible inventory from Rundeck nodes. So the main purpose of this PR is to support other Rundeck resource models (eg. Foreman) by generating temporary inventory files.
Origin of this PR are changes from #97 - we used it for a while, but it got closed.

### CODE REVIEW

**Comment by qvallance** [comment]
Thank you!  This is a key feature we've been waiting for.   Hoping this get reviewed and merged ASAP.

**Review by frozenice — APPROVED** [review]

**Comment by frozenice** [comment]
Thanks @erikasm! I invited you as a collaborator. Feel free to merge this if you feel confident. ;)
Check out the CONTRIBUTING.md file on how to publish a new version!

**Comment by mcassaniti** [comment]
@erikasm Is there something else you're waiting on for this PR? You're welcome to merge yourself, but I'm leaving it to you to merge.

**Pull request merged**
